src/routes/Verify/verifyView.scss

```diff
@@ -0,0 +1,75 @@
+@import 'styles/mixins';
+
+.verify-view {
+  all: initial;
+
+  @include box;
+  @include flex-column-center;
+
+  position: relative;
+  width: 100%;
+  height: 100%;
+
+  > .header {
```

Comment:
Seems, it is not used now?

Reply:
Yep, copy-pasted from another view.

But as soon as we add smth of value here we *may* need to copy-paste it again.

I think it is better to clean up after actual implementation.